### PR TITLE
Add create and edit steps to config file topic

### PR DESCRIPTION
**Pull request by shannonxtreme**
This change updates the structure of the config file Manage Secrets page. Detailed delta includes:

1. Line 22 - minor wording change for accuracy (define the object in the file, then create it)
2. Line 25 - changed "Secret data" to "the same data"
3. Add numbers to the steps
4. Combine the commands for base64 encoding to consolidate them.
1. Move note about -n flag to under the encode commands (moved existing content from old line 67-74 and trimmed down a bit)
1. Only change to the config file YAML is the password field, everything else is the same as before
2. Move old line 76-84 to new section about StringData to keep the actual create steps simple
1. Delete old line 121-183 (Check the Secret). This content will exist in only the kubectl page (as it is a kubectl command and output and the commands are the same)
1. Add section for editing the Secret by using a config file (new line 119-161)

This is one of three PRs that will make the "Manage Secrets using X" topics consistent to provide Create, Edit, and Delete steps for kubectl, config files, and kustomize. Original, larger PR showing the end state: https://github.com/kubernetes/website/pull/34674

This PR is part of a much larger updating of the Secrets documentation. The tracking issue is https://github.com/kubernetes/website/issues/4491.

/sig docs
/language en
/do-not-merge work-in-progress
/cc @tengqm

### CODE REVIEW

**Comment by shannonxtreme** [comment]
@tengqm do you think a line by line breakdown of the delta like in this PR's description will be helpful for the other two topics as well, to ease localization team's load?

File: content/en/docs/tasks/configmap-secret/managing-secret-using-config-file.md [hunk 1]
```diff
@@ -104,95 +109,67 @@ stringData:
...
-Where `YWRtaW5pc3RyYXRvcg==` decodes to `administrator`.
-
-## Clean Up
-
-To delete the Secret you have created:
+To delete a Secret, use `kubectl`:
 
 ```shell
-kubectl delete secret mysecret
+kubectl delete secret <secret-name>
```
Comment (tengqm) [inline content/en/docs/tasks/configmap-secret/managing-secret-using-config-file.md:168]: The original text looks good. As a task, we walk the readers through a process with a concrete example. In this task, we showed how to create a Secret named `mysecret` and now we show how to clean up.

This generalization is not needed.

File: content/en/docs/tasks/configmap-secret/managing-secret-using-config-file.md [hunk 2]
```diff
@@ -7,81 +7,86 @@ description: Creating Secret objects using resource configuration file.
...
+      name: mysecret
+    type: Opaque
+    data:
+      username: YWRtaW4=
+      password: VkxjNUAjdHltJEh6Qm5AOA==
+    ```
+
+    The name of the `Secret` object must be a valid
+    [DNS subdomain
+    name](/docs/concepts/overview/working-with-objects/names#dns-subdomain-names).
```
Comment (tengqm) [inline content/en/docs/tasks/configmap-secret/managing-secret-using-config-file.md:67]: Please keep hyperlinks on the same line. We are still unable to parse multiline links when identifying dangling ones.

File: content/en/docs/tasks/configmap-secret/managing-secret-using-config-file.md [hunk 3]
```diff
@@ -7,81 +7,86 @@ description: Creating Secret objects using resource configuration file.
...
-A practical example of this might be where you are deploying an application
-that uses a Secret to store a configuration file, and you want to populate
-parts of that configuration file during your deployment process.
-
-For example, if your application uses the following configuration file:
+## Create a Secret {#create-the-config-file}
+
+You can define the `Secret` object in a JSON or YAML configuration file before
+you create the object. The [Secret](/docs/reference/generated/kubernetes-api/{{<
+param "version" >}}/#secret-v1-core) resource contains two maps: `data` and
```
Comment (tengqm) [inline content/en/docs/tasks/configmap-secret/managing-secret-using-config-file.md:23]: Please keep hyperlinks on the same line. We are still unable to parse multiline links when identifying dangling ones.

File: content/en/docs/tasks/configmap-secret/managing-secret-using-config-file.md [hunk 4]
```diff
@@ -7,81 +7,86 @@ description: Creating Secret objects using resource configuration file.
...
+    ```
+
+    The name of the `Secret` object must be a valid
+    [DNS subdomain
+    name](/docs/concepts/overview/working-with-objects/names#dns-subdomain-names).
+
+1.  Apply the `Secret` object to your cluster:
+
+    ```shell
+    kubectl apply -f <path-to-configuration-file>
```
Comment (tengqm) [inline content/en/docs/tasks/configmap-secret/managing-secret-using-config-file.md:72]: I know someone are unhappy with the "configuration file" term.

In a task page, we expect users to copy/paste the examples to try each step. So I don't think we want to generalize this command.

File: content/en/docs/tasks/configmap-secret/managing-secret-using-config-file.md [hunk 5]
```diff
@@ -7,81 +7,86 @@ description: Creating Secret objects using resource configuration file.
...
+
+    ```shell
+    echo -n 'admin' | base64
+    echo -n 'VLc5@#tym$HzBn@8' | base64
+    ```
+    The `-n` flag ensures that there's no newline character at the end of your
+    data. If you're using the `base64` utility on Darwin/macOS, avoid using the
+    `-b` option to split long lines. Conversely, if you're on Linux, add the
+    `-w 0` option to `base64` commands or the pipeline `base64 | tr -d '\n'` if
+    the `-w` option is not available.
```
Comment (tengqm) [inline content/en/docs/tasks/configmap-secret/managing-secret-using-config-file.md:43]: In the original text, this paragraph was called out to be a note. It is something users may safely skip unless they have encountered some problems.

**Review by tengqm — COMMENTED** [review]
To be honest, the value of this rewriting is not clear as expected.

File: content/en/docs/tasks/configmap-secret/managing-secret-using-config-file.md (hunk 1 above)
Comment (shannonxtreme) [reply content/en/docs/tasks/configmap-secret/managing-secret-using-config-file.md:168]: Okay, I can change it back to the original example

**Comment by shannonxtreme** [comment]
I'm going to close this as it's now ported to #35383 and #35384

**Pull request closed without merging**